src/js/api/auth/login.test.js

```diff
@@ -0,0 +1,51 @@
+import { login } from './login';
+import { LocalStorageMock } from '../../storage/localStorageMock';
+
+const validEmail = 'student@stud.noroff.no';
+const validpassword = 'password123';
+const fakeApiResponse = {
+  name: 'User Name',
+  email: validEmail,
+  accessToken: 'sometoken28289jhjh',
+};
+
+global.localStorage = new LocalStorageMock();
```

Comment:
```suggestion
global.localStorage = new LocalStorageMock();
```

This could moved to a beforeAll hook: https://jestjs.io/docs/setup-teardown